pkg/operator/sync.go

```diff
@@ -34,17 +34,31 @@ func (optr *Operator) syncAll(config *OperatorConfig) error {
 				// return the outer error.
 				glog.Errorf("Error syncing ClusterOperatorStatus: %v", err)
 			}
-			glog.Errorf("Error syncing cluster api controller: %v", err)
+			glog.Errorf("Error syncing machine-api-controller: %v", err)
 			return err
 		}
-		glog.V(3).Info("Synced up all components")
+		glog.V(3).Info("Synced up all machine-api-controller components")
 	}
-
 	if err := optr.statusAvailable(); err != nil {
 		glog.Errorf("Error syncing ClusterOperatorStatus: %v", err)
 		return fmt.Errorf("error syncing ClusterOperatorStatus: %v", err)
 	}
 
+        // In addition, if the Provider is BareMetal, then bring up
```

Comment:
this should go before reporting available status

Reply:
Done.